llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -953,6 +953,12 @@ def FeatureRequiredExportPriority : SubtargetFeature<"required-export-priority",
   "Export priority must be explicitly manipulated on GFX11.5"
 >;
 
+def FeatureVmemWriteVgprInOrder : SubtargetFeature<"vmem-write-vgpr-in-order",
```

Comment:
Wouldn't it be easier to have a "VmemWriteVgprOutOfOrder" feature and just apply it to GFX12?

Reply:
"Easier" how? You mean it would make the patch smaller? I prefer to have features that state things in a "positive" way, so that not having the feature still generates conservatively correct code.

Reply:
Right, I didn't see things that way. I agree conservative is better